app/examples/react-16/src/App.js

```diff
@@ -1,12 +1,13 @@
 import React from 'react'
 
 import {
-  TimeSeriesStaticTest,
+  // TimeSeriesStaticTest,
```

Comment:
Why are we keeping this commented code here?

Reply:
The idea is just uncommenting/commenting it when it's necessary for testing manually